VMEncryption/main/CryptMountConfigUtil.py

```diff
@@ -353,7 +353,7 @@ def add_crypt_item_to_crypttab(self, crypt_item, backup_folder=None):
                 fstab_backup_line = None
                 with open("/etc/fstab") as f:
                     for line in f.readlines():
-                        device, mountpoint, fs = self.parse_fstab_line(line)
+                        device, mountpoint, fs, opts = self.parse_fstab_line(line)
```

Comment:
actual: opts
expected: fstab_opts (like other changes?)

Reply:
Just keeping it similar to other nearby variable names (mountpoint, device, etc). I feel given the context, it's descriptive enough